committeeoversightapp/models.py

```diff
@@ -101,6 +101,47 @@ def is_subcommittee(self):
         else:
             return False
 
+    @property
+    def is_permanent(self):
+        if self in CommitteeOrganization.objects.permanent_committees():
+            print("True")
+            return True
+        else:
+            return False
+
+    @property
+    def get_linked_html(self):
+        if self.is_subcommittee:
+            if self.parent_proxy.is_permanent:
+                return '<a href=\"{0}\">{1}</a>, {2}'.format(
+                    self.parent_proxy.url,
+                    self.parent,
+                    self
+                )
+            else:
+                return '{0}, {1}'.format(self.parent, self)
+        else:
+            if self.is_permanent:
+                return '<a href="{0}">{1}</a>'.format(self.url, self)
+            else:
+                return self
```

Comment:
`self`, not `str(self)`?

Reply:
Good catch. For future clarity I switched this and other similar occurrences to `self.name`